### PR TITLE
VxScan: Remove warning when ballot box is full

### PR DESCRIPTION
**Pull request by jonahkagan**
## Overview

Closes: #5136 

This is no longer a concern due to hardware improvements.
## Demo Video or Screenshot
N/A
## Testing Plan
Updated automated tests
## Checklist

- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.

- [ ] I have added a screenshot and/or video to this PR to demo the change
- [ ] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

File: apps/scan/frontend/src/screens/setup_scanner_screen.tsx [hunk 1]
```diff
@@ -1,25 +0,0 @@
-import { CenteredLargeProse, H1, P, appStrings } from '@votingworks/ui';
-import { ScreenMainCenterChild } from '../components/layout';
-
-interface Props {
-  scannedBallotCount?: number;
-}
-
-export function SetupScannerScreen({ scannedBallotCount }: Props): JSX.Element {
```
Comment (kshen0) [inline apps/scan/frontend/src/screens/setup_scanner_screen.tsx:8]: Is this related to the ballot bag? Looks like it's just an old file that was replaced by `internal_connection_problem_screen`?

**Review by kshen0 — APPROVED** [review]

File: apps/scan/frontend/src/screens/setup_scanner_screen.tsx (hunk 1 above)
Comment (jonahkagan) [reply apps/scan/frontend/src/screens/setup_scanner_screen.tsx:8]: No, it's not related. I just found that it was unused when I was using knip to make sure I had removed everything related to the ballot bag feature.
